src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorConfigurationService.cs

```diff
@@ -54,7 +55,7 @@ public async override Task<RazorLSPOptions> GetLatestOptionsAsync(CancellationTo
                 };
 
                 var response = _server.SendRequest("workspace/configuration", request);
-                var result = await response.Returning<object[]>(cancellationToken);
+                var result = await response.Returning<JObject[]>(CancellationToken.None);
```

Comment:
Why not use the CancellationToken which was already passed to this method?

Reply:
Good catch 👍 . I did this for testing. Will revert.